mon-pix/translations/fr.json

```diff
@@ -984,7 +984,7 @@
     },
     "fill-in-certificate-verification-code": {
       "title": "Vérifier un certificat Pix",
-      "description": "La certification Pix atteste d’un niveau de maitrise des compétences numériques, reconnu par l’État et le monde professionnel.",
+      "description": "La certification Pix atteste d’un niveau de maîtrise des compétences numériques : saisissez ci-après le \"code de vérification\" du certificat Pix à vérifier.",
```

Comment:
Il n'y a pas de tests qui pétouillent pour la modification de wording ?

Reply:
Non parce que les tests utilisent la clé mais pas la trad, je n'ai pas mis à jour

Reply:
mon-pix/tests/acceptance/fill-in-certificate-verification-code_test.js

Ça n'a pas l'air d'être testé même avec la clé.

Reply:
Heu...j'ai pas retrouvé 🫣